Example/Tests/GenerateUrlTests/UrlTests.swift

```diff
@@ -89,61 +89,68 @@ class UrlTests: BaseTestCase {
-        let config = CLDConfiguration(cloudName: "test123", apiKey: "a", apiSecret: "b", privateCdn: true, secure: true, secureDistribution: "something.cloudfront.net")
+        let config = CLDConfiguration(cloudName: "test123", apiKey: "a", apiSecret: "b", privateCdn: true, secure: true, secureDistribution: "something.cloudfront.net", analytics: false)
         sut = CLDCloudinary(configuration: config)
         let url = sut?.createUrl().generate("test")
         XCTAssertEqual(url, "https://something.cloudfront.net/image/upload/test")
     }
 
     func testHttpPrivateCdn() {
-        let config = CLDConfiguration(cloudName: "test123", apiKey: "a", apiSecret: "b", privateCdn: true)
+        let config = CLDConfiguration(cloudName: "test123", apiKey: "a", apiSecret: "b", privateCdn: true, analytics: false)
         sut = CLDCloudinary(configuration: config)
         let url = sut?.createUrl().generate("test")
         XCTAssertEqual(url, "http://test123-res.cloudinary.com/image/upload/test")
     }
 
     func testCdnSubDomain() {
-        let config = CLDConfiguration(cloudName: "test123", apiKey: "a", apiSecret: "b", cdnSubdomain: true)
+        let config = CLDConfiguration(cloudName: "test123", apiKey: "a", apiSecret: "b", cdnSubdomain: true, analytics: false)
         sut = CLDCloudinary(configuration: config)
         let url = sut?.createUrl().generate("test")
         XCTAssertEqual(url, "http://res-2.cloudinary.com/test123/image/upload/test")
     }
 
     func testSecureCdnSubDomainFalse() {
-        let config = CLDConfiguration(cloudName: "test123", apiKey: "a", apiSecret: "b", secure: true, cdnSubdomain: true)
+        let config = CLDConfiguration(cloudName: "test123", apiKey: "a", apiSecret: "b", secure: true, cdnSubdomain: true, analytics: false)
         sut = CLDCloudinary(configuration: config)
         let url = sut?.createUrl().generate("test")
         XCTAssertEqual(url, "\(prefix)/image/upload/test")
     }
 
     func testSecureCdnSubDomainTrue() {
-        let config = CLDConfiguration(cloudName: "test123", apiKey: "a", apiSecret: "b", privateCdn: true, secure: true, cdnSubdomain: true, secureCdnSubdomain: true)
+        let config = CLDConfiguration(cloudName: "test123", apiKey: "a", apiSecret: "b", privateCdn: true, secure: true, cdnSubdomain: true, secureCdnSubdomain: true, analytics: false)
         sut = CLDCloudinary(configuration: config)
         let url = sut?.createUrl().generate("test")
         XCTAssertEqual(url, "https://test123-res-2.cloudinary.com/image/upload/test")
     }
 
+    func testAnalyticsTrue() {
+        let config = CLDConfiguration(cloudName: "test123", apiKey: "a", apiSecret: "b", privateCdn: true, secure: true, cdnSubdomain: true, secureCdnSubdomain: true, analytics: true)
+        sut = CLDCloudinary(configuration: config)
+        let url = sut?.createUrl().generate("test")
+        XCTAssertEqual(url, "https://test123-res-2.cloudinary.com/image/upload/test?a_AEAEvAF0")
```

Comment:
Will this test work when swift and SDK versions change?

Reply:
No, should set them explicitly, will fix